packages/headless/src/controllers/search-parameter-manager/headless-search-parameter-manager.ts

```diff
@@ -119,8 +119,9 @@ function getCq(state: Partial<SearchParametersState>) {
   }
 
   const cq = state.advancedSearchQueries.cq;
-  const shouldInclude = cq !== getAdvancedSearchQueriesInitialState().cq;
-  return shouldInclude ? {cq} : {};
+  const shouldInclude =
+    cq !== null && cq !== getAdvancedSearchQueriesInitialState().cq;
```

Comment:
I'm uncertain whether I should or shouldn't include this null check since `getAdvancedSearchQueriesInitialState().cq` is null too.

The purpose of this condition is essentially just not to include `cq` if nothing ever set it.

Reply:
I prefer not comparing the `cq` to the initial request since the `initialState` feels pretty representative of what should be the initial constant query, and one or more requests could theoretically be sent before this happens (whether because of an implementation or our own changes).

I like the idea of a `initialConstantQuery` reducer though, that seems like it would work well in any situation and be easier to fix if something comes up. I'll add that.